src/css/ChatMap.css

```diff
@@ -13,9 +6,12 @@
 }
 
 .leaflet-control-zoom {
-    position: absolute;
-    top: 696px;
-    left: 533px;
+    /* position: absolute;
+    top: 655px;
+    left: 50%; */
+    /* bottom: 50px; */
+    /* transform: translate(-50%, 0); */
```

Comment:
comments?

Reply:
Can we leave this for task TAL-55 Search by map and I'll move this styles to ChatMapStyled and clear file?